### PR TITLE
update forge-electron and rebase project on webpack

### PR DESCRIPTION
**Pull request by manuel-calavera**
electon compile is deprecated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,19 +4,14 @@
...
-    "lint": "npm-run-all --serial pretty eslint",
-    "make": "run-s fetch-install-init electron-forge:make",
-    "package": "run-s fetch-install-init electron-forge:package",
-    "pretty": "prettier --write **/*.jsx **/*.js *.js .*.js",
-    "publish": "run-s fetch-install-init electron-forge:publish",
-    "start": "electron-forge start"
+    "package": "run-s fetch-install-init && electron-forge package",
+    "make": "electron-forge make",
+    "publish": "electron-forge publish",
+    "lint": "echo \"No linting configured\""
```
Comment (mvines) [inline package.json]: Why remove eslint?

File: package.json [hunk 2]
```diff
@@ -4,19 +4,14 @@
...
-    "electron-forge:make": "electron-forge make",
-    "electron-forge:package": "electron-forge package",
-    "electron-forge:publish": "electron-forge publish",
-    "eslint": "eslint --cache --color --ext .jsx,.js .",
+    "start": "electron-forge start",
     "fetch-install-init": "babel-node fetch-solana-install-init.js",
-    "lint": "npm-run-all --serial pretty eslint",
-    "make": "run-s fetch-install-init electron-forge:make",
-    "package": "run-s fetch-install-init electron-forge:package",
-    "pretty": "prettier --write **/*.jsx **/*.js *.js .*.js",
```
Comment (mvines) [inline package.json]: Why remove prettier?

File: forge.config.js [hunk 3]
```diff
@@ -21,29 +21,65 @@ const electronWinstallerConfig = {
...
+    [
+      '@electron-forge/plugin-webpack',
+      {
+        mainConfig: './webpack.main.config.js',
+        renderer: {
+          config: './webpack.renderer.config.js',
+          entryPoints: [
+            {
+              html: './src/index.html',
+              js: './src/renderer.jsx',
```
Comment (mvines) [inline forge.config.js]: This file doesn't seem to exist

**Comment by manuel-calavera** [comment]
@mvines well, this is how i split commits. let me merge more changes in this pr.

**Comment by mvines** [comment]
Well really I'd rather you go the other way!   "update forge-electron" is one thing, "rebase project on webpack" is another.  One PR for each thing.   Piling everything into a single PR is what I want to avoid.

**Comment by mvines** [comment]
Also you mention "electon compile is deprecated" as justification for "update forge-electron" (I assume?), but why "rebase project on webpack".

**Pull request closed without merging**